docs/docsite/rst/guide_aci.rst

```diff
@@ -0,0 +1,298 @@
+- `username` -- User name used to log on to the APIC *(defaults to `admin`)*
+- `password` -- Password for `username` to log on to the APIC (using password-based authentication)
+- `private_key` -- Private key for `username` to log on to APIC (using signature-based authentication)
+- `certificate_name` -- Name of the certificate in the ACI Web GUI *(defaults to `private_key` file baseename)*
+- `validate_certs` -- Validate certificate when using HTTPS communication *(defaults to `yes`)*
+- `use_ssl` -- Use HTTPS or HTTP for APIC REST communication *(defaults to `yes`)*
+- `use_proxy` -- Use system proxy settings *(defaults to `yes`)*
+- `timeout` -- Timeout value for socket-level communication
+
+
+ACI authentication
+------------------
+
+Password-based authentication
+.............................
+If you want to logon using a username and password, you can use the following parameters with your ACI modules:
+
+.. code-block:: yaml
+
+    username: admin
+    password: my-password
+
+Password-based authentication is very simple to work with, but it is not the most efficient form of authentication from ACI's point-of-view as it requires a separate login-request and an open session to work. To avoid having your session time-out and requiring another login, you can use the more efficient Signature-based authentication.
+
+**NOTE:** Password-based authentication also may trigger anti-DoS measures in ACI v3.1+ that causes session throttling and results in HTTP 503 errors and login failures. 
+
+
+Signature-based authentication using certificates
+.................................................
+Using signature-based authentication is more efficient and more reliable than password-based authentication.
+
+Generate certificate and private key
+,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+Signature-based authentication requires a (self-signed) X.509 certificate with private key, and a configuration step for your AAA user in ACI. To generate a working X.509 certificate and private key, use the following procedure:
+
+.. code-block:: bash
+
+    $ openssl req -new -newkey rsa:1024 -days 36500 -nodes -x509 -keyout admin.key -out admin.crt -subj '/CN=Admin/O=Your Company/C=US'
+
+Configure your local user
+,,,,,,,,,,,,,,,,,,,,,,,,,
+Then add the X.509 certificate to your ACI AAA local user at **ADMIN > AAA**. Then click **AAA Authentication** and check that in the **Authentication** field the **Realm** field displays **Local**. Expand **Security Management > Local Users**
+
+Click the name of the user you want to add a certificate to, in the **User Certificates** area, click the **+** sign and in the **Create X509 Certificate** enter a certificate name in the **Name** field (If you use the basename of your private key here, you don't need to enter **certificate_name** in Ansible) and copy&paste your X.509 certificate in the **Data** field.
+
+Obviously you can automate this by using the following Ansible task:
```

Comment:
`You can automate this by using the following Ansible task:`

Reply:
Done.